README.md

```diff
@@ -92,14 +92,14 @@ to ensure that CI workloads run in staging cannot interfere with production reso
 Yes. Currently, you can instantiate a new project from the stack and copy relevant components
 into your existing project to productionize it. The stack is modularized, so
 you can e.g. copy just the GitHub Actions workflows under `.github` or ML resource configs
- under `{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{cookiecutter.project_name_alphanumeric_underscore}}/databricks-resources` 
-and `{{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{cookiecutter.project_name_alphanumeric_underscore}}/bundle.yml` into your existing project.
+ under `{cookiecutter.root_dir__update_if_you_intend_to_use_monorepo}}/{{template `project_name_alphanumeric_underscore` .}}/databricks-resources` 
```

Comment:
I think the `{` and `}` are not matching, we have two `}` and one `{`}

Reply:
For easier review, there will be another PR to update all the docs.
This change is the side-effect of a string replacement targeting at the generated project.

Reply:
The docs PR and the test PR will be merged to this PR once they are approved.

Reply:
actually, let me revert this file.